packages/framework-provider-local/src/library/read-model-adapter.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  BoosterConfig,
+  Filter,
+  Logger,
+  ReadModelEnvelope,
+  ReadModelInterface,
+  UUID,
+} from '@boostercloud/framework-types'
+import { ReadModelRegistry } from '../services/read-model-registry'
+
+export async function rawReadModelEventsToEnvelopes(
+  config: BoosterConfig,
+  logger: Logger,
+  rawEvents: Array<unknown>
+): Promise<Array<ReadModelEnvelope>> {
+  return rawEvents.map((event) => event as ReadModelEnvelope)
+}
+
+export async function fetchReadModel(
+  db: ReadModelRegistry,
+  _config: BoosterConfig,
+  logger: Logger,
+  readModelName: string,
+  readModelID: UUID
+): Promise<ReadModelInterface> {
+  const response = await db.query({ typeName: readModelName, value: { id: readModelID } })
+  const item = response[0]
+  if (!item) {
+    console.log(`[ReadModelAdapter#fetchReadModel] Failed to fetch read model ${readModelName} with ID ${readModelID}`)
```

Comment:
I would not say `Failed`, as it is not a failure not to find a read model. I would say something like "not found".

Reply:
Done here https://github.com/boostercloud/booster/pull/473/commits/ca68b39d2f681f3d2d67629f2a1fc95075b32f8c